Peaky.Tests/TestClasses/ParametrizedTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+
+namespace Peaky.Tests.TestClasses
+{
+    public class ParametrizedTest : IApplyToApplication, IParametrizedTestCases
```

Comment:
Spelling: Parameterized

Reply:
According to dictionary are both US English valid words https://en.oxforddictionaries.com/definition/parameterize